README.md

```diff
@@ -80,10 +80,10 @@ If you have an active interest and you want to get involved, you have real power
 - Join the [SIG Cluster Lifecycle](https://groups.google.com/forum/#!forum/kubernetes-sig-cluster-lifecycle) Google Group for access to documents and calendars.
 - Participate in the conversations on [Kubernetes Discuss][kubernetes discuss]
 - Provider implementers office hours (CAPI)
-    - Weekly on Wednesdays @ 10:00 PT on [Zoom](https://zoom.us/j/861487554)
+    - Weekly on Wednesdays @ 10:00 am PT (Pacific Time) on [Zoom](https://zoom.us/j/861487554)
     - Previous meetings: \[ [notes][notes] | [recordings][recordings] \]
 - Cluster API Provider GCP office hours (CAPG)
-    - Monthly on first Thursday @ 10:00pm PT / 17:00 UTC on [Zoom](https://zoom.us/j/96963829102?pwd=WjBZcmwvZFZsUU93aVZieUk1L3FnZz09)
+    - Monthly on first Thursday @ 09:00 am PT (Pacific Time) on [Zoom](https://zoom.us/j/96963829102?pwd=WjBZcmwvZFZsUU93aVZieUk1L3FnZz09)
```

Comment:
I removed the UTC counterpart as this can create confusion in the time of the year where there is a drift in daylight saving times between US and other countries.